spec/controllers/formality_levels_controller_spec.rb

```diff
@@ -14,8 +14,34 @@
     end
 
     context "show" do
-      before { get :show, id: formality_level.id }
-      it { should respond_with :success }
+      context 'requesting standard representation' do
```

Comment:
Prefer double-quoted strings unless you need single quotes to avoid extra backslashes for escaping.